recipes/texlive-core/build.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig:$PREFIX/share/pkgconfig"
+
+mkdir -p tmp_build && pushd tmp_build
+  ../configure --prefix=$PREFIX \
+               --with-cairo=$PREFIX \
+               --datarootdir="$PREFIX/share/texlive" \
+               --disable-all-pkgs \
+               --disable-native-texlive-build \
+               --disable-ipc \
+               --disable-debug \
+               --disable-dependency-tracking \
+               --disable-mf \
+               --disable-pmp \
+               --disable-upmp \
+               --disable-aleph \
+               --disable-eptex \
+               --disable-euptex \
+               --disable-luatex \
+               --disable-luajittex \
+               --disable-uptex \
+               --enable-web2c \
+               --enable-silent-rules \
+               --enable-tex \
+               --enable-etex \
+               --enable-pdftex \
+               --enable-xetex \
+               --enable-web-progs \
+               --enable-texlive \
+               --enable-dvipdfm-x \
+               --with-icu=$PREFIX \
+               --with-mpfr=$PREFIX \
+               --with-gmp=$PREFIX \
+               --with-pixman=$PREFIX \
+               --with-freetype2=$PREFIX \
+               --with-libpng=$PREFIX \
+               --with-zlib=$PREFIX \
+               --x-libraries=$PREFIX/lib \
+               --x-includes=$PREFIX/include \
+               --without-system-harfbuzz \
+               --without-system-graphite2 \
+               --without-system-poppler
+
+  make
+  make check
```

Comment:
Based on this error message my recommendation is to change `make check` -> `LC_ALL=C make check`.